torch/utils/data/datapipes/utils/decoder.py

```diff
@@ -135,6 +132,8 @@ def __init__(self, imagespec):
         self.imagespec = imagespec.lower()
 
     def __call__(self, key, data):
+        import numpy as np
+        import PIL.Image
```

Comment:
I recommend wrapping it into 
```python
try:
  import
except:
 nice error
```

Reply:
Thanks! Will change it in the another PR!